make armc6: error 8 for assembly file

<!--

   ************************************** WARNING **************************************

   The ciarcom bot parses this header automatically. Any deviation from the 
   template may cause the bot to automatically correct this header or may result in a 
   warning message, requesting updates.

   Please ensure that nothing follows the Issue request type section, all 
   issue details are within the Description section and no changes are made to the 
   template format (as detailed below).

   *************************************************************************************

-->

### Description

<!--
    Required
    Add detailed description of what you are reporting.
    Good example: https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html
    Things to consider sharing:
    - What target does this relate to?
    - What toolchain (name + version) are you using?
    - What tools (name + version - is it mbed-cli, online compiler or IDE) are you using?
    - What is the SHA of Mbed OS (git log -n1 --oneline)?
    - Steps to reproduce. (Did you publish code or a test case that exhibits the problem?)
-->

Creating tracking issue for this as it seems no rootcase found so far.

Recent nightly and some PRs failed exporters job  (since yesterday as soon as make armc6 got actived in the exporters).

The failure indicates a problem with `mbed_config.h` file include in the assembly files. This is the log from CI:

```
==========STDERR==========
Warning: A3912W: Option 'c' is deprecated.
<built-in>:388:10: fatal error: 'mbed_config.h' file not found
#include "./mbed_config.h"
         ^~~~~~~~~~~~~~~~~
1 error generated.
Fatal error: A1905U: Pre-processor step failed for '../mbed-os/cmsis/TARGET_CORTEX_M/TOOLCHAIN_ARM/except.S'
1 Error, 1 Warning
make[1]: *** [mbed-os/cmsis/TARGET_CORTEX_M/TOOLCHAIN_ARM/except.o] Error 8
```

Interesting part, I was able to reproduce the same failure but only if I had armc5 in paths as well (and many other failures because of different armasm) - using windows. Our CI that is failing has only ARMC6 installed, it's linux based machine.

Locally, executed `python -u mbed-os/tools/test/examples/examples.py export make_armc6 --mcu K66F`

This is the error I get: 

```
process_begin: CreateProcess(NULL, C:/Program Files/Git/bin/sh.exe -c "echo \"main.o mbed-os/cmsis/TARGET_CORTEX_M/TOOLCHAIN_ARM/except.o mbed-os/cmsis/TARGET_CORTEX_M/mbed_fault_handler.o mbed-os/cmsis/TARGET_CORTEX_M/mbed_tz_context.o mbed-os/components/802.15.4_RF/atmel-rf-driver/source/NanostackRfPhyAtmel.o mbed-os/components/802.15.4_RF/atmel-rf-driver/source/at24mac.o mbed-os/components/802.15.4_RF/mcr20a-rf-driver/source/MCR20Drv.o mbed-os/components/802.15.4_RF/mcr20a-rf-driver/source/NanostackRfPhyMcr20a.o mbed-os/components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.o mbed-os/components/802.15.4_RF/stm-s2lp-rf-driver/source/rf_configuration.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_EMUL/psa_attest_inject_key.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_EMUL/psa_initial_attestation_api.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attest_boot_status_loader.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attest_crypto.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attest_crypto_keys.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attest_iat_claims_loader.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attestation_bootloader_data.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/psa_attestation_stubs.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/psa_inject_attestation_key_impl.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl/attest_token.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl/attestation_core.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl/t_cose/src/t_cose_sign1_sign.o mbed-os/components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl/t_cose/src/t_cose_util.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/src/UsefulBuf.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/src/ieee754.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/src/qcbor_decode.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/src/qcbor_encode.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/UsefulBuf_Tests.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/float_tests.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/half_to_double_from_rfc7049.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/qcbor_decode_tests.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/qcbor_encode_tests.o mbed-os/components/TARGET_PSA/services/attestation/qcbor/test/run_tests.o mbed-os/components/TARGET_PSA/services/platform/COMPONENT_PSA_SRV_EMUL/platform_emul.o mbed-os/components/TARGET_PSA/services/platform/COMPONENT_PSA_SRV_IMPL/platform_srv_impl.o mbed-os/components/TARGET_PSA/services/storage/common/psa_storage_common_impl.o mbed-os/components/TARGET_PSA/services/storage/its/COMPONENT_PSA_SRV_EMUL/psa_prot_internal_storage.o mbed-os/components/TARGET_PSA/services/storage/its/COMPONENT_PSA_SRV_IMPL/pits_impl.o mbed-os/components/TARGET_PSA/services/storage/ps/COMPONENT_NSPE/protected_storage.o mbed-os/components/storage/blockdevice/COMPONENT_FLASHIAP/COMMON/fslittle_test.o mbed-os/components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.o mbed-os/components/storage/blockdevice/COMPONENT_SD/SDBlockDevice.o mbed-os/components/wifi/esp8266-driver/ESP8266/ESP8266.o mbed-os/components/wifi/esp8266-driver/ESP8266Interface.o mbed-os/drivers/AnalogIn.o mbed-os/drivers/BusIn.o mbed-os/drivers/BusInOut.o mbed-os/drivers/BusOut.o mbed-os/drivers/CAN.o mbed-os/drivers/Ethernet.o mbed-os/drivers/FlashIAP.o mbed-os/drivers/I2C.o mbed-os/drivers/I2CSlave.o mbed-os/drivers/InterruptIn.o mbed-os/drivers/InterruptManager.o mbed-os/drivers/MbedCRC.o mbed-os/drivers/QSPI.o mbed-os/drivers/RawSerial.o mbed-os/drivers/SPI.o mbed-os/drivers/SPISlave.o mbed-os/drivers/Serial.o mbed-os/drivers/SerialBase.o mbed-os/drivers/TableCRC.o mbed-os/drivers/Ticker.o mbed-os/drivers/Timeout.o mbed-os/drivers/Timer.o mbed-os/drivers/TimerEvent.o mbed-os/drivers/UARTSerial.o mbed-os/events/EventQueue.o mbed-os/events/equeue/equeue.o mbed-os/events/equeue/equeue_mbed.o mbed-os/events/equeue/equeue_posix.o mbed-os/events/mbed_shared_queues.o mbed-os/features/cellular/framework/AT/ATHandler.o mbed-os/features/cellular/framework/AT/ATHandler_factory.o mbed-os/features/cellular/framework/AT/AT_CellularBase.o mbed-os/features/cellular/framework/AT/AT_CellularContext.o mbed-os/features/cellular/framework/AT/AT_CellularDevice.o mbed-os/features/cellular/framework/AT/AT_CellularInformation.o mbed-os/features/cellular/framework/AT/AT_CellularNetwork.o mbed-os/features/cellular/framework/AT/AT_CellularSMS.o mbed-os/features/cellular/framework/AT/AT_CellularStack.o mbed-os/features/cellular/framework/AT/AT_ControlPlane_netif.o mbed-os/features/cellular/framework/common/CellularLog.o mbed-os/features/cellular/framework/common/CellularUtil.o mbed-os/features/cellular/framework/device/CellularContext.o mbed-os/features/cellular/framework/device/CellularDevice.o mbed-os/features/cellular/framework/device/CellularStateMachine.o mbed-os/features/cellular/framework/targets/GEMALTO/CINTERION/GEMALTO_CINTERION.o mbed-os/features/cellular/framework/targets/GEMALTO/CINTERION/GEMALTO_CINTERION_CellularContext.o mbed-os/features/cellular/framework/targets/GEMALTO/CINTERION/GEMALTO_CINTERION_CellularInformation.o mbed-os/features/cellular/framework/targets/GEMALTO/CINTERION/GEMALTO_CINTERION_CellularStack.o mbed-os/features/cellular/framework/targets/GENERIC/GENERIC_AT3GPP/GENERIC_AT3GPP.o mbed-os/features/cellular/framework/targets/MultiTech/DragonflyNano/PPP/SARA4_PPP.o mbed-os/features/cellular/framework/targets/MultiTech/DragonflyNano/PPP/SARA4_PPP_CellularNetwork.o mbed-os/features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95.o mbed-os/features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95_CellularContext.o mbed-os/features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95_CellularInformation.o mbed-os/features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95_CellularNetwork.o mbed-os/features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95_CellularStack.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularContext.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularInformation.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularNetwork.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularStack.o mbed-os/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_ControlPlane_netif.o mbed-os/features/cellular/framework/targets/QUECTEL/M26/QUECTEL_M26.o mbed-os/features/cellular/framework/targets/QUECTEL/M26/QUECTEL_M26_CellularContext.o mbed-os/features/cellular/framework/targets/QUECTEL/M26/QUECTEL_M26_CellularInformation.o mbed-os/features/cellular/framework/targets/QUECTEL/M26/QUECTEL_M26_CellularStack.o mbed-os/features/cellular/framework/targets/QUECTEL/UG96/QUECTEL_UG96.o mbed-os/features/cellular/framework/targets/QUECTEL/UG96/QUECTEL_UG96_CellularContext.o mbed-os/features/cellular/framework/targets/TELIT/HE910/TELIT_HE910.o mbed-os/features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.o mbed-os/features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularContext.o mbed-os/features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularNetwork.o mbed-os/features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.o mbed-os/features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP.o mbed-os/features/device_key/source/DeviceKey.o mbed-os/features/frameworks/greentea-client/source/greentea_metrics.o mbed-os/features/frameworks/greentea-client/source/greentea_serial.o mbed-os/features/frameworks/greentea-client/source/greentea_test_env.o mbed-os/features/frameworks/mbed-client-cli/source/ns_cmdline.o mbed-os/features/frameworks/mbed-client-randlib/source/randLIB.o mbed-os/features/frameworks/mbed-coap/source/sn_coap_builder.o mbed-os/features/frameworks/mbed-coap/source/sn_coap_header_check.o mbed-os/features/frameworks/mbed-coap/source/sn_coap_parser.o mbed-os/features/frameworks/mbed-coap/source/sn_coap_protocol.o mbed-os/features/frameworks/mbed-trace/source/mbed_trace.o mbed-os/features/frameworks/nanostack-libservice/source/IPv6_fcf_lib/ip_fsc.o mbed-os/features/frameworks/nanostack-libservice/source/libBits/common_functions.o mbed-os/features/frameworks/nanostack-libservice/source/libList/ns_list.o mbed-os/features/frameworks/nanostack-libservice/source/libip4string/ip4tos.o mbed-os/features/frameworks/nanostack-libservice/source/libip4string/stoip4.o mbed-os/features/frameworks/nanostack-libservice/source/libip6string/ip6tos.o mbed-os/features/frameworks/nanostack-libservice/source/libip6string/stoip6.o mbed-os/features/frameworks/nanostack-libservice/source/nsdynmemLIB/nsdynmemLIB.o mbed-os/features/frameworks/nanostack-libservice/source/nvmHelper/ns_nvm_helper.o mbed-os/features/frameworks/unity/source/unity.o mbed-os/features/frameworks/utest/mbed-utest-shim.o mbed-os/features/frameworks/utest/source/unity_handler.o mbed-os/features/frameworks/utest/source/utest_case.o mbed-os/features/frameworks/utest/source/utest_default_handlers.o mbed-os/features/frameworks/utest/source/utest_greentea_handlers.o mbed-os/features/frameworks/utest/source/utest_harness.o mbed-os/features/frameworks/utest/source/utest_shim.o mbed-os/features/frameworks/utest/source/utest_stack_trace.o mbed-os/features/frameworks/utest/source/utest_types.o mbed-os/features/lorawan/LoRaWANInterface.o mbed-os/features/lorawan/LoRaWANStack.o mbed-os/features/lorawan/lorastack/mac/LoRaMac.o mbed-os/features/lorawan/lorastack/mac/LoRaMacChannelPlan.o mbed-os/features/lorawan/lorastack/mac/LoRaMacCommand.o mbed-os/features/lorawan/lorastack/mac/LoRaMacCrypto.o mbed-os/features/lorawan/lorastack/phy/LoRaPHY.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYAS923.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYAU915.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYCN470.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYCN779.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYEU433.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYEU868.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYIN865.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYKR920.o mbed-os/features/lorawan/lorastack/phy/LoRaPHYUS915.o mbed-os/features/lorawan/system/LoRaWANTimer.o mbed-os/features/lwipstack/LWIPInterface.o mbed-os/features/lwipstack/LWIPInterfaceEMAC.o mbed-os/features/lwipstack/LWIPInterfaceL3IP.o mbed-os/features/lwipstack/LWIPMemoryManager.o mbed-os/features/lwipstack/LWIPStack.o mbed-os/features/lwipstack/lwip-sys/arch/lwip_checksum.o mbed-os/features/lwipstack/lwip-sys/arch/lwip_memcpy.o mbed-os/features/lwipstack/lwip-sys/arch/lwip_sys_arch.o mbed-os/features/lwipstack/lwip-sys/lwip_random.o mbed-os/features/lwipstack/lwip-sys/lwip_tcp_isn.o mbed-os/features/lwipstack/lwip/src/api/lwip_api_lib.o mbed-os/features/lwipstack/lwip/src/api/lwip_api_msg.o mbed-os/features/lwipstack/lwip/src/api/lwip_err.o mbed-os/features/lwipstack/lwip/src/api/lwip_netbuf.o mbed-os/features/lwipstack/lwip/src/api/lwip_netdb.o mbed-os/features/lwipstack/lwip/src/api/lwip_netifapi.o mbed-os/features/lwipstack/lwip/src/api/lwip_sockets.o mbed-os/features/lwipstack/lwip/src/api/lwip_tcpip.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_autoip.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_dhcp.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_etharp.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_icmp.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_igmp.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_ip4.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_ip4_addr.o mbed-os/features/lwipstack/lwip/src/core/ipv4/lwip_ip4_frag.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_dhcp6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_ethip6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_icmp6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_inet6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_ip6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_ip6_addr.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_ip6_frag.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_mld6.o mbed-os/features/lwipstack/lwip/src/core/ipv6/lwip_nd6.o mbed-os/features/lwipstack/lwip/src/core/lwip_def.o mbed-os/features/lwipstack/lwip/src/core/lwip_dns.o mbed-os/features/lwipstack/lwip/src/core/lwip_inet_chksum.o mbed-os/features/lwipstack/lwip/src/core/lwip_init.o mbed-os/features/lwipstack/lwip/src/core/lwip_ip.o mbed-os/features/lwipstack/lwip/src/core/lwip_mem.o mbed-os/features/lwipstack/lwip/src/core/lwip_memp.o mbed-os/features/lwipstack/lwip/src/core/lwip_netif.o mbed-os/features/lwipstack/lwip/src/core/lwip_pbuf.o mbed-os/features/lwipstack/lwip/src/core/lwip_raw.o mbed-os/features/lwipstack/lwip/src/core/lwip_stats.o mbed-os/features/lwipstack/lwip/src/core/lwip_sys.o mbed-os/features/lwipstack/lwip/src/core/lwip_tcp.o mbed-os/features/lwipstack/lwip/src/core/lwip_tcp_in.o mbed-os/features/lwipstack/lwip/src/core/lwip_tcp_out.o mbed-os/features/lwipstack/lwip/src/core/lwip_timeouts.o mbed-os/features/lwipstack/lwip/src/core/lwip_udp.o mbed-os/features/lwipstack/lwip/src/netif/lwip_ethernet.o mbed-os/features/lwipstack/lwip/src/netif/lwip_ethernetif.o mbed-os/features/lwipstack/lwip/src/netif/lwip_lowpan6.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_auth.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_ccp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_chap-md5.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_chap-new.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_chap_ms.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_demand.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_eap.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_ecp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_eui64.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_fsm.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_ipcp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_ipv6cp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_lcp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_magic.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_mppe.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_multilink.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_ppp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_pppapi.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_pppcrypt.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_pppoe.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_pppol2tp.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_pppos.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_upap.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_utils.o mbed-os/features/lwipstack/lwip/src/netif/ppp/lwip_vj.o mbed-os/features/lwipstack/lwip/src/netif/ppp/polarssl/lwip_arc4.o mbed-os/features/lwipstack/lwip/src/netif/ppp/polarssl/lwip_des.o mbed-os/features/lwipstack/lwip/src/netif/ppp/polarssl/lwip_md4.o mbed-os/features/lwipstack/lwip/src/netif/ppp/polarssl/lwip_md5.o mbed-os/features/lwipstack/lwip/src/netif/ppp/polarssl/lwip_sha1.o mbed-os/features/lwipstack/lwip_tools.o mbed-os/features/lwipstack/ppp_lwip.o mbed-os/features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto.o mbed-os/features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto_slot_management.o mbed-os/features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto_storage.o mbed-os/features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto_storage_file.o mbed-os/features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto_storage_its.o mbed-os/features/mbedtls/platform/TARGET_PSA/COMPONENT_NSPE/src/psa_hrng.o mbed-os/features/mbedtls/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/src/default_random_seed.o mbed-os/features/mbedtls/platform/src/mbed_trng.o mbed-os/features/mbedtls/platform/src/platform_alt.o mbed-os/features/mbedtls/src/aes.o mbed-os/features/mbedtls/src/aesni.o mbed-os/features/mbedtls/src/arc4.o mbed-os/features/mbedtls/src/aria.o mbed-os/features/mbedtls/src/asn1parse.o mbed-os/features/mbedtls/src/asn1write.o mbed-os/features/mbedtls/src/base64.o mbed-os/features/mbedtls/src/bignum.o mbed-os/features/mbedtls/src/blowfish.o mbed-os/features/mbedtls/src/camellia.o mbed-os/features/mbedtls/src/ccm.o mbed-os/features/mbedtls/src/certs.o mbed-os/features/mbedtls/src/chacha20.o mbed-os/features/mbedtls/src/chachapoly.o mbed-os/features/mbedtls/src/cipher.o mbed-os/features/mbedtls/src/cipher_wrap.o mbed-os/features/mbedtls/src/cmac.o mbed-os/features/mbedtls/src/ctr_drbg.o mbed-os/features/mbedtls/src/debug.o mbed-os/features/mbedtls/src/des.o mbed-os/features/mbedtls/src/dhm.o mbed-os/features/mbedtls/src/ecdh.o mbed-os/features/mbedtls/src/ecdsa.o mbed-os/features/mbedtls/src/ecjpake.o mbed-os/features/mbedtls/src/ecp.o mbed-os/features/mbedtls/src/ecp_curves.o mbed-os/features/mbedtls/src/entropy.o mbed-os/features/mbedtls/src/entropy_poll.o mbed-os/features/mbedtls/src/error.o mbed-os/features/mbedtls/src/gcm.o mbed-os/features/mbedtls/src/havege.o mbed-os/features/mbedtls/src/hkdf.o mbed-os/features/mbedtls/src/hmac_drbg.o mbed-os/features/mbedtls/src/md.o mbed-os/features/mbedtls/src/md2.o mbed-os/features/mbedtls/src/md4.o mbed-os/features/mbedtls/src/md5.o mbed-os/features/mbedtls/src/md_wrap.o mbed-os/features/mbedtls/src/memory_buffer_alloc.o mbed-os/features/mbedtls/src/net_sockets.o mbed-os/features/mbedtls/src/nist_kw.o mbed-os/features/mbedtls/src/oid.o mbed-os/features/mbedtls/src/padlock.o mbed-os/features/mbedtls/src/pem.o mbed-os/features/mbedtls/src/pk.o mbed-os/features/mbedtls/src/pk_wrap.o mbed-os/features/mbedtls/src/pkcs11.o mbed-os/features/mbedtls/src/pkcs12.o mbed-os/features/mbedtls/src/pkcs5.o mbed-os/features/mbedtls/src/pkparse.o mbed-os/features/mbedtls/src/pkwrite.o mbed-os/features/mbedtls/src/platform.o mbed-os/features/mbedtls/src/platform_util.o mbed-os/features/mbedtls/src/poly1305.o mbed-os/features/mbedtls/src/ripemd160.o mbed-os/features/mbedtls/src/rsa.o mbed-os/features/mbedtls/src/rsa_internal.o mbed-os/features/mbedtls/src/sha1.o mbed-os/features/mbedtls/src/sha256.o mbed-os/features/mbedtls/src/sha512.o mbed-os/features/mbedtls/src/ssl_cache.o mbed-os/features/mbedtls/src/ssl_ciphersuites.o mbed-os/features/mbedtls/src/ssl_cli.o mbed-os/features/mbedtls/src/ssl_cookie.o mbed-os/features/mbedtls/src/ssl_srv.o mbed-os/features/mbedtls/src/ssl_ticket.o mbed-os/features/mbedtls/src/ssl_tls.o mbed-os/features/mbedtls/src/threading.o mbed-os/features/mbedtls/src/timing.o mbed-os/features/mbedtls/src/version.o mbed-os/features/mbedtls/src/version_features.o mbed-os/features/mbedtls/src/x509.o mbed-os/features/mbedtls/src/x509_create.o mbed-os/features/mbedtls/src/x509_crl.o mbed-os/features/mbedtls/src/x509_crt.o mbed-os/features/mbedtls/src/x509_csr.o mbed-os/features/mbedtls/src/x509write_crt.o mbed-os/features/mbedtls/src/x509write_csr.o mbed-os/features/mbedtls/src/xtea.o mbed-os/features/mbedtls/targets/hash_wrappers.o mbed-os/features/nanostack/coap-service/source/coap_connection_handler.o mbed-os/features/nanostack/coap-service/source/coap_message_handler.o mbed-os/features/nanostack/coap-service/source/coap_security_handler.o mbed-os/features/nanostack/coap-service/source/coap_service_api.o mbed-os/features/nanostack/mbed-mesh-api/source/CallbackHandler.o mbed-os/features/nanostack/mbed-mesh-api/source/LoWPANNDInterface.o mbed-os/features/nanostack/mbed-mesh-api/source/MeshInterfaceNanostack.o mbed-os/features/nanostack/mbed-mesh-api/source/NanostackEMACInterface.o mbed-os/features/nanostack/mbed-mesh-api/source/NanostackEthernetInterface.o mbed-os/features/nanostack/mbed-mesh-api/source/NanostackMemoryManager.o mbed-os/features/nanostack/mbed-mesh-api/source/ThreadInterface.o mbed-os/features/nanostack/mbed-mesh-api/source/WisunInterface.o mbed-os/features/nanostack/mbed-mesh-api/source/ethernet_tasklet.o mbed-os/features/nanostack/mbed-mesh-api/source/mesh_system.o mbed-os/features/nanostack/mbed-mesh-api/source/nd_tasklet.o mbed-os/features/nanostack/mbed-mesh-api/source/thread_tasklet.o mbed-os/features/nanostack/mbed-mesh-api/source/wisun_tasklet.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_fhss_timer.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_interrupt.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_random.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_timer.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/cs_nvm/cs_nvm.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/ns_event_loop.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/ns_event_loop_mbed.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/ns_event_loop_mutex.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/ns_hal_init.o mbed-os/features/nanostack/nanostack-hal-mbed-cmsis-rtos/nvm/nvm_ram.o mbed-os/features/nanostack/nanostack-interface/Nanostack.o mbed-os/features/nanostack/sal-stack-nanostack-eventloop/source/event.o mbed-os/features/nanostack/sal-stack-nanostack-eventloop/source/ns_timeout.o mbed-os/features/nanostack/sal-stack-nanostack-eventloop/source/ns_timer.o mbed-os/features/nanostack/sal-stack-nanostack-eventloop/source/system_timer.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Bootstraps/Generic/network_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Bootstraps/Generic/protocol_6lowpan.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Bootstraps/Generic/protocol_6lowpan_bootstrap.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Bootstraps/Generic/protocol_6lowpan_interface.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Fragmentation/cipv6_fragmenter.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/IPHC_Decode/6lowpan_iphc.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/IPHC_Decode/iphc_compress.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/IPHC_Decode/iphc_decompress.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/IPHC_Decode/lowpan_context.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/beacon_handler.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/mac_data_poll.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/mac_helper.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/mac_ie_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/mac_pairwise_key.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/MAC/mac_response_handler.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Mesh/mesh.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ND/nd_router_object.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/NVM/nwk_nvm.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_bbr_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_beacon.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_bootstrap.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_border_router_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_commissioning_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_commissioning_if.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_common.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_dhcpv6_server.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_diagnostic.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_discovery.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_host_bootstrap.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_joiner_application.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_leader_service.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_lowpower_private_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_management_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_management_client.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_management_if.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_management_server.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_mdns.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_meshcop_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_mle_message_handler.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_nd.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_neighbor_class.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_net_config_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_network_data_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_network_data_storage.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_network_synch.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_nvm_store.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_resolution_client.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_resolution_server.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_router_bootstrap.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_routing.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_test_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/adaptation_interface.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_bbr_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_bootstrap.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_common.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_eapol_auth_relay.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_eapol_pdu.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_eapol_relay.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_eapol_relay_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_empty_functions.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_ie_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_llc_data_service.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_management_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_mpx_header.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_neighbor_class.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_auth.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_controller.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_nvm_data.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_nvm_store.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_pae_supp.o mbed-os/features/nanostack/sal-stack-nanostack/source/6LoWPAN/ws/ws_test_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/BorderRouter/border_router.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/icmpv6.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/icmpv6_prefix.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/icmpv6_radv.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/ipv6.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/ipv6_flow.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/ipv6_fragmentation.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/ipv6_resolution.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/mld.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/tcp.o mbed-os/features/nanostack/sal-stack-nanostack/source/Common_Protocols/udp.o mbed-os/features/nanostack/sal-stack-nanostack/source/Core/buffer_dyn.o mbed-os/features/nanostack/sal-stack-nanostack/source/Core/ns_address_internal.o mbed-os/features/nanostack/sal-stack-nanostack/source/Core/ns_socket.o mbed-os/features/nanostack/sal-stack-nanostack/source/Core/sockbuf.o mbed-os/features/nanostack/sal-stack-nanostack/source/DHCPv6_Server/DHCPv6_Server_service.o mbed-os/features/nanostack/sal-stack-nanostack/source/DHCPv6_client/dhcpv6_client_service.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_fhss_callbacks.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_filter.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_header_helper_functions.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_indirect_data.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_mcps_sap.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_mlme.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_pd_sap.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_security_mib.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/mac_timer.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/IEEE802_15_4/sw_mac.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/ethernet/ethernet_mac_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/rf_driver_storage.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/serial/serial_mac_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/virtual_rf/virtual_rf_client.o mbed-os/features/nanostack/sal-stack-nanostack/source/MAC/virtual_rf/virtual_rf_driver.o mbed-os/features/nanostack/sal-stack-nanostack/source/MLE/mle.o mbed-os/features/nanostack/sal-stack-nanostack/source/MLE/mle_tlv.o mbed-os/features/nanostack/sal-stack-nanostack/source/MPL/mpl.o mbed-os/features/nanostack/sal-stack-nanostack/source/NWK_INTERFACE/protocol_core.o mbed-os/features/nanostack/sal-stack-nanostack/source/NWK_INTERFACE/protocol_core_sleep.o mbed-os/features/nanostack/sal-stack-nanostack/source/NWK_INTERFACE/protocol_stats.o mbed-os/features/nanostack/sal-stack-nanostack/source/NWK_INTERFACE/protocol_timer.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_control.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_data.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_downward.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_mrhof.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_objective.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_of0.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_policy.o mbed-os/features/nanostack/sal-stack-nanostack/source/RPL/rpl_upward.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/Common/security_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/eap_protocol.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_avp.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_client.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_eap_header.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_header.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_relay_table.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/PANA/pana_server.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/TLS/tls_ccm_crypt.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/TLS/tls_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/eapol/eapol_helper.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/eapol/kde_helper.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/kmp/kmp_addr.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/kmp/kmp_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/kmp/kmp_eapol_pdu_if.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/kmp/kmp_socket_if.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/eap_tls_sec_prot/auth_eap_tls_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/eap_tls_sec_prot/eap_tls_sec_prot_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/eap_tls_sec_prot/supp_eap_tls_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/fwh_sec_prot/auth_fwh_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/fwh_sec_prot/supp_fwh_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/gkh_sec_prot/auth_gkh_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/gkh_sec_prot/supp_gkh_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/key_sec_prot/key_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/sec_prot_certs.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/sec_prot_keys.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/sec_prot_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/tls_sec_prot/tls_sec_prot.o mbed-os/features/nanostack/sal-stack-nanostack/source/Security/protocols/tls_sec_prot/tls_sec_prot_lib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/CCM_lib/ccm_security.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/CCM_lib/mbedOS/aes_mbedtls_adapter.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/Neighbor_cache/neighbor_cache.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/SHA256_Lib/ns_sha256.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/SHA256_Lib/shalib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/Trickle/trickle.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/blacklist/blacklist.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/etx/etx.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/channel_functions.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/channel_list.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_channel.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_common.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_configuration_interface.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_statistics.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_ws.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fhss/fhss_ws_empty_functions.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/fnv_hash/fnv_hash.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/hmac/hmac_sha1.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/ieee_802_11/ieee_802_11.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/load_balance/load_balance.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mac_neighbor_table/mac_neighbor_table.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/fnet/fnet_stack/services/mdns/fnet_mdns.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/fnet/fnet_stack/services/poll/fnet_poll.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/fnet/fnet_stack/stack/fnet_stdlib.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/ns_fnet_events.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/ns_fnet_port.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/ns_mdns_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mle_service/mle_service.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mle_service/mle_service_buffer.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mle_service/mle_service_frame_counter_table.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mle_service/mle_service_interface.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/mle_service/mle_service_security.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/nd_proxy/nd_proxy.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/nist_aes_kw/nist_aes_kw.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/pan_blacklist/pan_blacklist.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/utils/isqrt.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/utils/ns_crc.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/utils/ns_file_system.o mbed-os/features/nanostack/sal-stack-nanostack/source/Service_Libs/whiteboard/whiteboard.o mbed-os/features/nanostack/sal-stack-nanostack/source/ipv6_stack/ipv6_routing_table.o mbed-os/features/nanostack/sal-stack-nanostack/source/ipv6_stack/protocol_ipv6.o mbed-os/features/nanostack/sal-stack-nanostack/source/libDHCPv6/dhcp_service_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/libDHCPv6/libDHCPv6.o mbed-os/features/nanostack/sal-stack-nanostack/source/libDHCPv6/libDHCPv6_server.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/multicast_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_6lowpan_parameter_api.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_ipv6.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_load_balance.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_mle.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_rpl.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_short_address_extension.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/net_test.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/ns_net.o mbed-os/features/nanostack/sal-stack-nanostack/source/libNET/src/socket_api.o mbed-os/features/netsocket/DTLSSocket.o mbed-os/features/netsocket/DTLSSocketWrapper.o mbed-os/features/netsocket/EMACInterface.o mbed-os/features/netsocket/EthernetInterface.o mbed-os/features/netsocket/InternetSocket.o mbed-os/features/netsocket/L3IPInterface.o mbed-os/features/netsocket/NetStackMemoryManager.o mbed-os/features/netsocket/NetworkInterface.o mbed-os/features/netsocket/NetworkInterfaceDefaults.o mbed-os/features/netsocket/NetworkStack.o mbed-os/features/netsocket/SocketAddress.o mbed-os/features/netsocket/SocketStats.o mbed-os/features/netsocket/TCPServer.o mbed-os/features/netsocket/TCPSocket.o mbed-os/features/netsocket/TLSSocket.o mbed-os/features/netsocket/TLSSocketWrapper.o mbed-os/features/netsocket/UDPSocket.o mbed-os/features/netsocket/WiFiAccessPoint.o mbed-os/features/netsocket/cellular/CellularNonIPSocket.o mbed-os/features/netsocket/emac-drivers/TARGET_Freescale_EMAC/TARGET_K66F/hardware_init_MK66F18.o mbed-os/features/netsocket/emac-drivers/TARGET_Freescale_EMAC/kinetis_emac.o mbed-os/features/netsocket/nsapi_dns.o mbed-os/features/nfc/acore/source/ac_buffer.o mbed-os/features/nfc/acore/source/ac_buffer_builder.o mbed-os/features/nfc/acore/source/ac_buffer_reader.o mbed-os/features/nfc/acore/source/ac_stream.o mbed-os/features/nfc/source/controllers/PN512Driver.o mbed-os/features/nfc/source/controllers/PN512SPITransportDriver.o mbed-os/features/nfc/source/controllers/PN512TransportDriver.o mbed-os/features/nfc/source/nfc/NFCController.o mbed-os/features/nfc/source/nfc/NFCControllerDriver.o mbed-os/features/nfc/source/nfc/NFCEEPROM.o mbed-os/features/nfc/source/nfc/NFCEEPROMDriver.o mbed-os/features/nfc/source/nfc/NFCNDEFCapable.o mbed-os/features/nfc/source/nfc/NFCRemoteEndpoint.o mbed-os/features/nfc/source/nfc/NFCRemoteInitiator.o mbed-os/features/nfc/source/nfc/NFCTarget.o mbed-os/features/nfc/source/nfc/Type4RemoteInitiator.o mbed-os/features/nfc/source/nfc/ndef/MessageBuilder.o mbed-os/features/nfc/source/nfc/ndef/MessageParser.o mbed-os/features/nfc/source/nfc/ndef/RecordParser.o mbed-os/features/nfc/source/nfc/ndef/common/Mime.o mbed-os/features/nfc/source/nfc/ndef/common/SimpleMessageParser.o mbed-os/features/nfc/source/nfc/ndef/common/Text.o mbed-os/features/nfc/source/nfc/ndef/common/URI.o mbed-os/features/nfc/source/nfc/ndef/common/util.o mbed-os/features/nfc/stack/ndef/ndef.o mbed-os/features/nfc/stack/platform/nfc_scheduler.o mbed-os/features/nfc/stack/platform/nfc_transport.o mbed-os/features/nfc/stack/tech/iso7816/iso7816.o mbed-os/features/nfc/stack/tech/iso7816/iso7816_app.o mbed-os/features/nfc/stack/tech/isodep/isodep_target.o mbed-os/features/nfc/stack/tech/type4/type4_target.o mbed-os/features/nfc/stack/transceiver/pn512/pn512.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_cmd.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_hw.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_irq.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_poll.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_registers.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_rf.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_timer.o mbed-os/features/nfc/stack/transceiver/pn512/pn512_transceive.o mbed-os/features/nfc/stack/transceiver/transceiver.o mbed-os/features/storage/blockdevice/BufferedBlockDevice.o mbed-os/features/storage/blockdevice/ChainingBlockDevice.o mbed-os/features/storage/blockdevice/ExhaustibleBlockDevice.o mbed-os/features/storage/blockdevice/FlashSimBlockDevice.o mbed-os/features/storage/blockdevice/HeapBlockDevice.o mbed-os/features/storage/blockdevice/MBRBlockDevice.o mbed-os/features/storage/blockdevice/ObservingBlockDevice.o mbed-os/features/storage/blockdevice/ProfilingBlockDevice.o mbed-os/features/storage/blockdevice/ReadOnlyBlockDevice.o mbed-os/features/storage/blockdevice/SlicingBlockDevice.o mbed-os/features/storage/filesystem/Dir.o mbed-os/features/storage/filesystem/File.o mbed-os/features/storage/filesystem/FileSystem.o mbed-os/features/storage/filesystem/fat/ChaN/ff.o mbed-os/features/storage/filesystem/fat/ChaN/ffunicode.o mbed-os/features/storage/filesystem/fat/FATFileSystem.o mbed-os/features/storage/filesystem/littlefs/LittleFileSystem.o mbed-os/features/storage/filesystem/littlefs/littlefs/lfs.o mbed-os/features/storage/filesystem/littlefs/littlefs/lfs_util.o mbed-os/features/storage/kvstore/conf/kv_config.o mbed-os/features/storage/kvstore/direct_access_devicekey/DirectAccessDevicekey.o mbed-os/features/storage/kvstore/filesystemstore/FileSystemStore.o mbed-os/features/storage/kvstore/global_api/kvstore_global_api.o mbed-os/features/storage/kvstore/kv_map/KVMap.o mbed-os/features/storage/kvstore/securestore/SecureStore.o mbed-os/features/storage/kvstore/tdbstore/TDBStore.o mbed-os/features/storage/nvstore/source/nvstore.o mbed-os/features/storage/system_storage/SystemStorage.o mbed-os/hal/LowPowerTickerWrapper.o mbed-os/hal/mbed_critical_section_api.o mbed-os/hal/mbed_flash_api.o mbed-os/hal/mbed_gpio.o mbed-os/hal/mbed_itm_api.o mbed-os/hal/mbed_lp_ticker_api.o mbed-os/hal/mbed_lp_ticker_wrapper.o mbed-os/hal/mbed_pinmap_common.o mbed-os/hal/mbed_pinmap_default.o mbed-os/hal/mbed_ticker_api.o mbed-os/hal/mbed_us_ticker_api.o mbed-os/hal/mpu/mbed_mpu_v7m.o mbed-os/hal/mpu/mbed_mpu_v8m.o mbed-os/platform/ATCmdParser.o mbed-os/platform/CThunkBase.o mbed-os/platform/CallChain.o mbed-os/platform/FileBase.o mbed-os/platform/FileHandle.o mbed-os/platform/FilePath.o mbed-os/platform/FileSystemHandle.o mbed-os/platform/LocalFileSystem.o mbed-os/platform/Stream.o mbed-os/platform/mbed_alloc_wrappers.o mbed-os/platform/mbed_application.o mbed-os/platform/mbed_assert.o mbed-os/platform/mbed_board.o mbed-os/platform/mbed_critical.o mbed-os/platform/mbed_error.o mbed-os/platform/mbed_error_hist.o mbed-os/platform/mbed_interface.o mbed-os/platform/mbed_mem_trace.o mbed-os/platform/mbed_mktime.o mbed-os/platform/mbed_mpu_mgmt.o mbed-os/platform/mbed_poll.o mbed-os/platform/mbed_retarget.o mbed-os/platform/mbed_rtc_time.o mbed-os/platform/mbed_sdk_boot.o mbed-os/platform/mbed_semihost_api.o mbed-os/platform/mbed_sleep_manager.o mbed-os/platform/mbed_stats.o mbed-os/platform/mbed_wait_api_no_rtos.o mbed-os/platform/mbed_wait_api_rtos.o mbed-os/rtos/ConditionVariable.o mbed-os/rtos/EventFlags.o mbed-os/rtos/Kernel.o mbed-os/rtos/Mutex.o mbed-os/rtos/RtosTimer.o mbed-os/rtos/Semaphore.o mbed-os/rtos/TARGET_CORTEX/SysTimer.o mbed-os/rtos/TARGET_CORTEX/TOOLCHAIN_ARM_STD/mbed_boot_arm_std.o mbed-os/rtos/TARGET_CORTEX/mbed_boot.o mbed-os/rtos/TARGET_CORTEX/mbed_rtos_rtx.o mbed-os/rtos/TARGET_CORTEX/mbed_rtx_handlers.o mbed-os/rtos/TARGET_CORTEX/mbed_rtx_idle.o mbed-os/rtos/TARGET_CORTEX/rtx4/cmsis_os1.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Config/RTX_Config.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_ARM/TARGET_RTOS_M4_M7/irq_cm4f.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_delay.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_evflags.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_evr.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_kernel.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_lib.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_memory.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_mempool.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_msgqueue.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_mutex.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_semaphore.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_system.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_thread.o mbed-os/rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_timer.o mbed-os/rtos/TARGET_CORTEX/rtx5/Source/os_systick.o mbed-os/rtos/TARGET_CORTEX/rtx5/Source/os_tick_ptim.o mbed-os/rtos/ThisThread.o mbed-os/rtos/Thread.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/PeripheralPins.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/crc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/fsl_clock_config.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/fsl_phy.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/mbed_overrides.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/TOOLCHAIN_ARM_STD/startup_MK66F18.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/system_MK66F18.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_adc16.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_clock.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_cmp.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_cmt.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_crc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_dac.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_dmamux.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_dspi.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_dspi_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_enet.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_ewm.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_flash.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_flexbus.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_flexcan.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_ftm.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_gpio.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_i2c.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_i2c_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_llwu.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_lmem_cache.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_lptmr.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_lpuart.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_lpuart_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_pdb.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_pit.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_pmc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_rcm.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_rnga.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_rtc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sai.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sai_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sdhc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sdramc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sim.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_smc.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_sysmpu.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_tpm.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_tsi_v4.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_uart.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_uart_edma.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_vref.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/drivers/fsl_wdog.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/pwmout_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/serial_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/spi_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/trng_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/us_ticker.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/analogin_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/analogout_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/dma_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/flash_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/gpio_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/gpio_irq_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/i2c_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/lp_ticker.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/pinmap.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/port_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/qspi_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/rtc_api.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/sleep.o mbed-os/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/fsl_common.o mbed-os/usb/device/USBAudio/USBAudio.o mbed-os/usb/device/USBDevice/EndpointResolver.o mbed-os/usb/device/USBDevice/USBDevice.o mbed-os/usb/device/USBHID/USBHID.o mbed-os/usb/device/USBHID/USBKeyboard.o mbed-os/usb/device/USBHID/USBMouse.o mbed-os/usb/device/USBHID/USBMouseKeyboard.o mbed-os/usb/device/USBMIDI/USBMIDI.o mbed-os/usb/device/USBMSD/USBMSD.o mbed-os/usb/device/USBSerial/USBCDC.o mbed-os/usb/device/USBSerial/USBSerial.o mbed-os/usb/device/hal/mbed_usb_phy.o mbed-os/usb/device/targets/TARGET_Freescale/USBPhy_Kinetis.o mbed-os/usb/device/utilities/AsyncOp.o mbed-os/usb/device/utilities/ByteBuffer.o mbed-os/usb/device/utilities/LinkedListBase.o mbed-os/usb/device/utilities/OperationListBase.o mbed-os/usb/device/utilities/events/PolledQueue.o mbed-os/usb/device/utilities/events/TaskBase.o\" > .link_options.txt", ...) failed.
make (e=87): The parameter is incorrect.
make[1]: *** [mbed-os-example-blinky.elf] Error 87
make: *** [makefile:26: all] Error 2
```


Same error if I run just `make` in the example blinky (using make 4.2 or 3.81)

Too long command?

@SenRamakri @ARMmbed/mbed-os-test 

### Issue request type

<!--
    Required
    Please add only one X to one of the following types. Do not fill multiple types (split the issue otherwise.)
    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
    description heading and to add a 'x' to the correct box.
-->
    [ ] Question
    [ ] Enhancement
    [X] Bug

--- a/tools/export/makefile/__init__.py
+++ b/tools/export/makefile/__init__.py
@@ -18,7 +18,7 @@ from __future__ import print_function, absolute_import
 from builtins import str
 
 from os.path import splitext, basename, relpath, join, abspath, dirname,\
-    exists
+    exists, normpath
 from os import remove
 import sys
 from subprocess import check_output, CalledProcessError, Popen, PIPE
@@ -38,6 +38,15 @@ SHELL_ESCAPE_TABLE = {
 def shell_escape(string):
     return "".join(SHELL_ESCAPE_TABLE.get(char, char) for char in string)
 
+def _fix_include_asm_flag(flag, ctx):
+    if flag.startswith('-I'):
+        new_path = normpath(join(ctx['vpath'][0], flag[2:]))
+        return "-I{}".format(new_path)
+    elif flag.startswith('--preinclude='):
+        new_path = normpath(join(ctx['vpath'][0], flag[13:]))
+        return "--preinclude={}".format(new_path)
+    else:
+        return flag
 
 class Makefile(Exporter):
     """Generic Makefile template that mimics the behavior of the python build
@@ -139,12 +148,14 @@ class Makefile(Exporter):
         # Add the virtual path the the include option in the ASM flags
         new_asm_flags = []
         for flag in ctx['asm_flags']:
-            if flag.startswith('-I'):
-                new_asm_flags.append("-I{}/{}".format(ctx['vpath'][0], flag[2:]))
-            elif flag.startswith('--preinclude='):
-                new_asm_flags.append("--preinclude={}/{}".format(ctx['vpath'][0], flag[13:]))
+            if flag.startswith('--cpreproc_opts='):
+                sub_flags = flag.split(',')
+                new_sub_flags = []
+                for sub_flag in sub_flags:
+                    new_sub_flags.append(_fix_include_asm_flag(sub_flag, ctx))
+                new_asm_flags.append(','.join(new_sub_flags))
             else:
-                new_asm_flags.append(flag)
+                new_asm_flags.append(_fix_include_asm_flag(flag, ctx))
         ctx['asm_flags'] = new_asm_flags
 
         for templatefile in \
@@ -273,33 +284,37 @@ class Armc5(Arm):
     NAME = 'Make-ARMc5'
     TOOLCHAIN = "ARM"
     PREPROCESS_ASM = True
-    
+
     @classmethod
     def is_target_supported(cls, target_name):
         target = TARGET_MAP[target_name]
-                
         if int(target.build_tools_metadata["version"]) > 0:
             #Although toolchain name is set to ARM above we should check for ARMC5 for 5.12/onwards
             if "ARMC5" not in target.supported_toolchains:
                 return False
-        
-        return apply_supported_whitelist(
-            cls.TOOLCHAIN, cls.POST_BINARY_WHITELIST, target)
+
+        arm_res = apply_supported_whitelist(
+            "ARM", cls.POST_BINARY_WHITELIST, target
+        )
+        armc5_res = apply_supported_whitelist(
+            "ARMC5", cls.POST_BINARY_WHITELIST, target
+        )
+        return arm_res or armc5_res
 
 class Armc6(Arm):
     """ARM Compiler 6 (armclang) specific generic makefile target"""
     NAME = 'Make-ARMc6'
     TOOLCHAIN = "ARMC6"
-    
+
     @classmethod
     def is_target_supported(cls, target_name):
         target = TARGET_MAP[target_name]
-        
+
         if int(target.build_tools_metadata["version"]) > 0:
             if not (len(set(target.supported_toolchains).intersection(
                     set(["ARM", "ARMC6"]))) > 0):
                 return False
-        
+
             if not apply_supported_whitelist(
                 cls.TOOLCHAIN, cls.POST_BINARY_WHITELIST, target):
                 #ARMC6 is not in the list, but also check for ARM as ARM represents ARMC6 for 5.12/onwards
@@ -311,7 +326,7 @@ class Armc6(Arm):
         else:
             return apply_supported_whitelist(
                     cls.TOOLCHAIN, cls.POST_BINARY_WHITELIST, target)
-                
+
 
 class IAR(Makefile):
     """IAR specific makefile target"""

